configs/twins/README.md

```diff
@@ -0,0 +1,38 @@
+# Twins
+
+> [Twins: Revisiting the Design of Spatial Attention in Vision Transformers](https://arxiv.org/pdf/2104.13840.pdf)
```

Comment:
Please use the abstract link instead pdf llink.

Reply:
done.